pkg/operator2/operator.go

```diff
@@ -122,11 +126,12 @@ func NewAuthenticationOperator(
 
 		oauthClientClient: oauthClientClient.OAuthClients(),
 
-		services:    kubeClient.CoreV1(),
-		endpoints:   kubeClient.CoreV1(),
-		secrets:     kubeClient.CoreV1(),
-		configMaps:  kubeClient.CoreV1(),
-		deployments: kubeClient.AppsV1(),
+		kubeClientHolder: resourceapply.NewKubeClientHolder(kubeClient),
```

Comment:
you'll want to wire in the informers for the new resources

Reply:
the configmap added is the `v4-0-config-system-trusted-ca-bundle` and the branding secret is `v4-0-config-system-ocp-branding-template` so those are covered by the `getPrefixFilter()` function, right? https://github.com/openshift/cluster-authentication-operator/blob/master/pkg/operator2/operator.go#L170-#L171

Reply:
yep, those should already be covered :)

Reply:
but you are still missing an informer for the NS of the operand, the rolebinding and the SA

Reply:
added all but clusterrolebinding, mentioned in note below